server/errorstore/errors.go

```diff
@@ -87,10 +83,12 @@ func runHandler(ctx context.Context, eh *Handler) {
 	go eh.handleErrors(ctx)
 }
 
-// Flush retrieves all stored errors from Redis and returns them as a slice of
-// JSON-encoded strings. It is a destructive read - the errors are removed from
-// Redis on return.
-func (h *Handler) Flush() ([]string, error) {
+// Retrieve retrieves all stored errors from Redis and returns them as a slice
+// of JSON-encoded strings.
+//
+// If flush is provied performs a destructive read - the errors are removed
+// from Redis on return.
+func (h *Handler) Retrieve(flush bool) ([]string, error) {
```

Comment:
should we use a struct here and in related methods that pass around the `flush` flag?

Reply:
IMO, this is a-ok as is.